drivers/modem/Kconfig.cellular

```diff
@@ -43,11 +43,10 @@ config MODEM_CELLULAR_UART_BUFFER_SIZES
 	default 512
 
 config MODEM_CELLULAR_CMUX_MAX_FRAME_SIZE
-	int "The maximum CMUX frame size in bytes."
-	default 32 if DT_HAS_U_BLOX_LARA_R6_ENABLED
-	default 128
+	int "[Deprecated] The maximum frame size in bytes."
+	default 0
```

Comment:
You should probably add here
```
select DEPRECATED
```
I think one does not have to write `[Deprecated]` to prompt as selecting the DEPRECATED will give user enough info.

Reply:
int values cannot select another value.

Reply:
:facepalm: Indeed, never mind then